book/tutorials/mental-health/index.md

```diff
@@ -0,0 +1,69 @@
+
+## Broad format
+
+Respecting everyone's time committment, we have designed this session to be semi-interactive. The session will be a mix of interview-style interaction between Anthony and Shashank, and include small group discussions between participants in small groups. 
+
+
+## Links to material
+
+In this section, we provide our initial curated list of resources. We acknowledge that this list is non-exhaustive, and encourage participants to suggest additional relevant resources with which they are familiar with.
+
+* Institution based mental health support: Many institutions have resources to support students and employees for their mental health needs. Here we provide examples for University of Washington students and employees as reference, but encourage participants for exploring similar resources at their respective institutions.
+    * Anyone in an immediate crisis situation should call or text 988, or chat on [988Lifeline.org](https://988Lifeline.org).  
+    * Students: All students at the University of Washington have access to same day crisis counselling through the [Husky Helpline](https://wellbeing.uw.edu/huskyhelpline/). 
+    * More structured support can be accessed through the [peer wellness coaching group](https://livewell.uw.edu/pwc/), [UW Mental health counselling center](https://wellbeing.uw.edu/unit/counseling-center/), and [group therapy programs](https://www.polisci.washington.edu/news/2023/01/19/advertised-groups-winter-2023). 
+    * Services by the Mental Health Counselling Center are all covered by student insurance (ISHIP or GAIP). We encourage participants to learn more about these resources [here](https://wellbeing.uw.edu/mental-health/mental-health-resources/).
+    * Faculty/Staff/Postdocs: Faculty and staff can avail short-term, focussed therapy sessions through the Washington State Employee Assistance Program ([WA EAP](https://hr.uw.edu/worklife/employee-assistance-program/)).
+    * Everyone enrolled with [PEBB](https://www.hca.wa.gov/pebb-benefits-admins/pebb-benefits#:~:text=The%20Public%20Employees%20Benefits%20Board,Care%20Assistance%20Program%20(DCAP).) is eligible.
+* Resources for workplace-related mental health issues relevant to geoscientists:
+    * [Climate Anxiety & Eco-Grief](https://www.drjenniferatkinson.com/)
+    * [climate pshchiatry alliance](https://www.climatepsychiatry.org/)
+    * [Workplace mental health and well-being](https://www.hhs.gov/surgeongeneral/priorities/workplace-well-being/index.html)
+    * [Work That Reconnects Network](https://www.hhs.gov/surgeongeneral/priorities/workplace-well-being/index.html)
+* Participating in supportive communities: Many of us struggle alone and have inhibtations to discuss our feelings because we might fail to identify in the same way as our collagues in the workplace. In these situations, finding communities and groups of scientists whom we can identify with personally, seek mentorship from, and rediscover purpose can be helpful. 
```

Comment:
```suggestion
* Participating in supportive and inclusive communities: Many of us struggle alone and have inhibitions when discussing our feelings because we might fail to identify in the same way as our everyday workplace colleagues. This is particularly relevant if we identify ourselves as marginalized in some form or another. In these situations, engaging with communities and groups of scientists we can identify with personally might help us seek mentorship and rediscover our purpose. Along these lines, we provide an initial list of organizations that have a core focus on inclusivity and mentorship:
```